app/views/hyrax/dashboard/collections/new.html.erb

```diff
@@ -1,6 +1,7 @@
-<% provide :page_title, construct_page_title( t('.header', type_title: @collection.collection_type.title) ) %>
+<% provide :page_title, construct_page_title( t('.header') ) %>
+
 <% provide :page_header do %>
-  <h1><span class="fa fa-edit"></span> <%= t('.header', type_title: @collection.collection_type.title) %> </h1>
+  <h1><span class="fa fa-edit"></span> <%= t('.header') %></h1>
```

Comment:
Not sure why this file is changed?  If you revert this back to what it is in collections-sprint branch, the failing feature test passes.

Reply:
I tried to revert it but grabbed the master version of this file by mistake. Correcting and pushing again in a minute!